elastic/shared/runners/datastream.py

```diff
@@ -50,6 +50,33 @@ async def create(es, params):
     return ops, "ops"
 
 
+class DeleteRemoteDataStream(Runner):
+    multi_cluster = True
+
+    def __init__(self):
+        super().__init__()
+
+    async def __call__(self, multi_es, params):
+        """
+        Deletes specified datastreams from all clusters
+        """
+        ops = 0
+        for cluster_name, cluster_client in multi_es.items():
+            data_stream = mandatory(params, "data-stream", "delete-datastream")
+            logger = logging.getLogger(__name__)
+            try:
+                logger.info("Deleting data stream: [%s] in cluster [%s]", data_stream, cluster_name)
+                await cluster_client.indices.delete_data_stream(name=data_stream)
+            except elasticsearch.ElasticsearchException as e:
+                msg = f"Failed to delete datastream [{data_stream}]; [{e}]"
+                raise BaseException(msg)
```

Comment:
trying to remember, is there something about pickle/thespian/serialization that means we shouldn't throw with this?
```suggestion
                raise BaseException(msg, e)
```

Reply:
Not that I'm aware of, it looks like we do properly catch these via https://github.com/elastic/rally/commit/19d4cfa21c494aac8c68eda5ce0ce52c623dc3ce